tools/ui_integration.md

```diff
@@ -0,0 +1 @@
+Looking for setup_ui_integration.py script? Try checking [here](https://github.com/DataBiosphere/jade-data-repo/tree/develop/tools/setupResourceScripts)
```

Comment:
Do you think this is helpful? I thought it might be nice to add a pointer where people expected the scripts to be.

Reply:
This is fine although maybe instead reference the PR that merged the files in? The path to the code could change over time so unless you pin the version in the path, it could get stale. A PR link would always exist. Or you could link to the repo as whole and let the reader look for the script by name.